cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/mazha/json/JSONHelper.java

```diff
@@ -104,4 +107,17 @@ public JavaType mapStringToObject(){
         return this.getTypeFactory().constructParametricType(Map.class,String.class,Object.class);
     }
 
+    public JavaType mapStringMissingRevisions(){
+        return this.getTypeFactory().constructParametricType(Map.class, String.class, CouchClient
```

Comment:
I still think we could make these static constants, but I'm happy for that to defer to some other time. In fact I think we could make this whole JSONHelper utility static instead of having loads of them.